pkg/adsc/adsc.go

```diff
@@ -204,6 +205,16 @@ type ResponseHandler interface {
 
 var adscLog = log.RegisterScope("adsc", "adsc debugging", 0)
 
+// ConvertProtoMessageToString help to implement the conversion from Proto message to string
+func ConvertProtoMessageToString(obj proto.Message) (string, error) {
+	jsonm := &jsonpb.Marshaler{EnumsAsInts: true, EmitDefaults: true, OrigName: true, Indent: "  "}
```

Comment:
This is not quite correct in all cases, although maybe all that matter here. ADSC can handle Istio/Kubernetes API types which use gogo protobuf which is not compatible with golang/protobuf/jsonpb

Reply:
Yes, u r correct,  I notice that, so I did not change all of the json marshaling. However, I will change a name for this function to avoid unnecessary misunderstand.

Reply:
Yes, u r correct,  I notice that, so I did not change all of the json marshaling. However, I will change a name for this function to avoid unnecessary misunderstand.

Reply:
Yes, u r correct,  I notice that, so I did not change all of the json marshaling. However, I will change a name for this function to avoid unnecessary misunderstand.

Reply:
done!